.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,52 +6,39 @@ labels: 'Type: Enhancement'
 ---
 
 <!--
-IMPORTANT NOTES
-
 Thank you for taking the time to suggest a feature. Before submitting,
 please reread your description to ensure that other readers can reasonably
 understand the motivation and proposed solution.
 
-This form is written in GitHub's Markdown format. For a reference on this type
-of syntax, see GitHub's documentation:
-https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
-
-This template contains guidance for your submission within the < ! - -, - - > blocks.
-These are comments in HTML syntax and will not appear in the submission.
-Be sure to use the "Preview" feature on GitHub to ensure your submission is formatted as intended.
-
-When including code snippets, please paste the text itself and wrap the code block with
-ticks (see the other character on the tilde ~ key in a US keyboard) to format it as code.
-For example, Python code should be wrapped in ticks like this:
-```python
-def a_func():
-    return 1
-
-a = 1
-b = a_func()
-print(a + b)
-```
-This is preferred over screen shots since it is searchable and others can copy/paste
-the text to run it.
+IMPORTANT NOTES
+
+1. Replace all example text (contained in "<>") or anywhere specifically commenting to replace the
```

Comment:
Do we really want them to replace the text? I usually just place my text under the instruction text and leave the instructions for quick reference during later edits.

Reply:
I think generally the `<!- >` blocks are more useful since they can be left in the raw file but do not render. Also, the replacement direction is a little confusing since `<!-- >` is a subset of `<>`. I would prefer to have all instructions left in for reference and just have users place their text under the instructions.

Reply:
I like the call out to replace the text by saying `< add text here>` but I agree with Jared we could move the descriptions to HTML comments if they don't say "replace text here" or "add text here" so the information for the specific section is maintained if you go back in later to add more details.

Reply:
I do like having the `<>` to show where to add text because I usually get confused with really long HTML comments of where I should add my information.

Reply:
I like where things are at now, but we should remove the comment about replacing "<>" blocks since it is irrelevant.

Reply:
I think the preview for this thread is just outdated now because I deleted that piece earlier today.